dspace-api/src/main/resources/org/dspace/storage/rdbms/sqlmigration/postgres/V7.6_2024.09.24__registration_data.sql

```diff
@@ -0,0 +1,52 @@
+--
```

Comment:
@vins01-4science : In my attempts to test this PR, I'm having difficulty getting this migration to automatically run in Docker because it's seen as "out of order" when compared to the other 9.0 migrations.  Since the migration doesn't run automatically, it results in Hibernate errors (because of missing columns in "registrationdata") which cause the Docker instance to error out (and stop running).

Could we consider renaming this to be `V9.0_2025.03.12__registration_data.sql`?  It also doesn't make sense to me that we'd name this migration "V7.6" because it will not be backported officially to DSpace 7.6 because this is a new feature.  The release where this feature will first be available will be 9.0.